docs/markdown/pipelines/exomeseq.md

```diff
@@ -42,106 +58,127 @@ Running WGS analysis with Sarek does not require any additional files.
 
 ### Quick start
 
-- Latest stable release `-r 2.7.1`.
+- Latest stable release `-r 3.1.2`.
 
 Please use these parameters to run your WGS analysis with sarek:
 
 ```bash
 #!/bin/bash
-nextflow run nf-core/sarek -r 2.7.1 \
+nextflow run nf-core/sarek -r 3.1.2 \
 -profile cfc \
 --genome 'GRCh38' \
---input 'input.tsv' \
---tools 'HaplotypeCaller,Strelka,mutect2,manta,ascat,controlFreec,VEP,snpEff' \
+--input 'input.csv' \
+--aligner 'bwa-mem2' \
+--tools 'haplotypecaller,strelka,mutect2,manta,ascat,controlfreec,cnvkit,vep,snpeff' \
 ```
 
-### Known issues
+> **_Note_**: We parallelize across genomic regions. This speeds up computation at the expense of storage. Since often storage is a bottleneck, it is useful to reduce the amount of parallelization. For this, you can increase the parameter `--nucleotides_per_second`. Recommended setting: `100000`.
 
-- There is currently a bug regarding the reference names: if you specify `--genome GRCh38`, sarek actually uses `hg38` (UCSC naming convention, and not Ensembl). See the [issue](https://github.com/nf-core/sarek/issues/86).
+### Somatic Variant Calling
 
-### Fixed issues since 2.6.1
+#### Paired
 
-- The `2.6.1` release had a bug with `mutect2`, this is now addressed and can be used.
-- Our cluster has problems with MarkDuplicates Spark, which is why is should be avoided. As of `2.7` the non-spark variant is run by default. Please be aware to set `--no_gatk_spark` if you run `2.6.1` or earlier.
+If your analysis requires _only_ somatic variant calls and no germline variant calling, set `--only_paired_variant_calling` to reduce the amount of computation done.
+
+#### Tumor-only
+
+If you run tumor-only samples without matched normals, it is _highly_ recommended to generate a panel-of-normals for Mutect2 and CNVKit. For details, see here for [mutect2](https://nf-co.re/sarek/3.1.2/usage#how-to-create-a-panel-of-normals-for-mutect2) and [cnvkit](https://cnvkit.readthedocs.io/en/stable/tumor.html) for details.
+
+#### Visualization
+
+Depending on your project, `vep`-annotated vcf makes creation of oncoplots with [maftools](https://bioconductor.org/packages/release/bioc/vignettes/maftools/inst/doc/maftools.html) easier.
 
 ## Whole Exome and Targeted Sequencing
 
 Running WES analysis and targeted sequencing with Sarek requires the BED file that specifies the Exomic regions or the Targeted regions that were used in the exome capture kit or Targeted design.
```

Comment:
Minor beauty suggestion: For uniformity reasons: write `sarek` with or without capital letters throughout the text (as it is a name maybe capital is the choice `Sarek`)